packages/vision/src/index.js

```diff
@@ -960,12 +960,19 @@ Vision.prototype.detectCrops = function(images, options, callback) {
  *   //     },
  *   //     confidence: 56.748849,
  *   //     anger: false,
+ *   //     angerLikelihood: 1,
```

Comment:
@rtw do you think we should go with the score here (`1`) or the string value, i.e. `UNLIKELY`?